git_machete/tests/functional/test_machete.py

```diff
@@ -2409,12 +2409,8 @@ def test_github_checkout_prs(self) -> None:
         repo: str
         org: str
         (org, repo) = get_parsed_github_remote_url(self.repo_sandbox.remote_path)
-        expected_error_message = f"PR #100 is not found in repository `{org}/{repo}`"
-        with self.assertRaises(MacheteException) as e:
-            self.launch_command('github', 'checkout-prs', '100')
-        if e:
-            self.assertEqual(e.exception.parameter, expected_error_message,
-                             'Verify that expected error message has appeared when given pull request to checkout does not exists.')
+        expected_msg = f"Warn: PR #100 is not found in repository `{org}/{repo}`\n"
```

Comment:
Pls add a test case for where `--mine`/`--by` would behave differently before & after this change in implementation